Essentials/src/com/earth2me/essentials/commands/Commandwhois.java

```diff
@@ -56,8 +56,9 @@ public void run(final Server server, final CommandSource sender, final String co
             sender.sendMessage(tl("whoisAFK", tl("false")));
         }
         sender.sendMessage(tl("whoisJail", (user.isJailed() ? user.getJailTimeout() > 0 ? DateUtil.formatDateDiff(user.getJailTimeout()) : tl("true") : tl("false"))));
-        sender.sendMessage(tl("whoisMuted", (user.isMuted() ? user.getMuteTimeout() > 0 ? DateUtil.formatDateDiff(user.getMuteTimeout()) : tl("true") : tl("false"))));
-
+        sender.sendMessage(user.hasMuteReason() ? tl("whoisMutedReason", (user.isMuted() ? user.getMuteTimeout() > 0 ?
+                DateUtil.formatDateDiff(user.getMuteTimeout()) : tl("true") : tl("false")),user.getMuteReason()) : tl("whoisMuted", (user.isMuted() ?
+                user.getMuteTimeout() > 0 ? DateUtil.formatDateDiff(user.getMuteTimeout()) : tl("true") : tl("false"))));
```

Comment:
There is an actual reason to expand these ternary statements into `if ... else`s here, as they are quite complex.